src/main/java/com/labsynch/labseer/service/StandardizationServiceImpl.java

```diff
@@ -467,71 +469,58 @@ private int dupeCheckStandardizationStructure(Long dryRunId) throws CmpdRegMolFo
 					chemStructureHashMap,
 					StructureType.STANDARDIZATION_DRY_RUN);
 
+			// Fetch the dry run dupes (aka new duplicates)
+
 			// Pass -1F for simlarityPercent (non nullable int required in function
 			// signature not used in DUPLICATE_TAUTOMER searches)
 			// Pass -1 for maxResults (non nullable int required in function signature we
 			// don't want to limit the hit counts here)
 			hits = chemStructureService.searchMolStructures(cmpdRegMolecules.get(tmpStructureKey),
 					StructureType.STANDARDIZATION_DRY_RUN, SearchType.DUPLICATE_TAUTOMER, -1F, -1);
-			newDupeCount = hits.length;
-			for (int hit : hits) {
```

Comment:
Removes all the logic that was for duplicate checking before for both the dry run structures (new dupes) and parent dupe checks (old dupes)